src/main/java/me/itzg/helpers/users/model/UserDef.java

```diff
@@ -0,0 +1,30 @@
+package me.itzg.helpers.users.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import lombok.Data;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.jackson.Jacksonized;
+
+@Data @SuperBuilder
+@Jacksonized
+public class UserDef {
+    final String name;
+    final List<String> flags;
+
+    public UserDef(String input) {
+        final int colonIndex = input.trim().indexOf(':');
+        if (colonIndex < 0) {
+            name = input.trim();
+            flags = new ArrayList<>();
+            return;
+        }
+        name = input.substring(0, colonIndex).trim();
+        flags = Arrays.stream(input.substring(colonIndex + 1).split(","))
+            .map(String::trim)
+            .filter(flag -> !flag.isEmpty())
+            .toList();
```

Comment:
Unfortunately this code has to be compiled source-compatible with Java 8 (thanks to Forge 1.12.x), so this needs to be

```suggestion
            .collect(Collectors.toList());
```

FYI I have the image build ready to pin helper version per java version, but still need to think that through.